distributions/Dist_hk1764.py

```diff
@@ -0,0 +1,47 @@
+
+import numpy as np
+from base_distribution import BaseDistribution
+
+class Dist_hk1764(BaseDistribution):
+	def __init__(self):
+		self.f_max = 1
+		self.x_min = -1
+		self.x_max = 1
+
+
+	def pdf(self, x):
+		"""This is your PDF"""
+		return np.abs((x**2-x**3)/x)
+
+	def mean(self):
+		"""This is the mean of the PDF"""
+		return -.41
```

Comment:
If I have done something wrong, I can re do it with different pdf if you would like

Best,
Hark
> On Apr 10, 2016, at 20:48, Kyle Cranmer <notifications@github.com> wrote:
> 
> In distributions/Dist_hk1764.py <https://github.com/cranmer/intro-exp-phys-II/pull/42#discussion_r59145396>:
> 
> > +from base_distribution import BaseDistribution
> > +
> > +class Dist_hk1764(BaseDistribution):
> > +	def __init__(self):
> > +		self.f_max = 1
> > +		self.x_min = -1
> > +		self.x_max = 1
> > +
> > +
> > +	def pdf(self, x):
> > +		"""This is your PDF"""
> > +		return np.abs((x**2-x**3)/x)
> > +
> > +	def mean(self):
> > +		"""This is the mean of the PDF"""
> > +		return -.41
> where does this come from? Did you use calculus to derive it?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/cranmer/intro-exp-phys-II/pull/42/files/bfd89479dfa283543a339489d72acb25522f0887#r59145396>